src/cmd/ship.go

```diff
@@ -96,31 +96,31 @@ func ensureParentBranchIsMainBranch(branchName string) {
 
 func getShipStepList(config shipConfig) (result steps.StepList) {
 	mainBranch := git.GetMainBranch()
-	areInitialAndTargetDifferent := config.TargetBranch != config.InitialBranch
+	areInitialAndBranchToShipDifferent := config.BranchToShip != config.InitialBranch
```

Comment:
I think this should be renamed to `areInitialBranchAndBranchToShipDifferent` in order to be consistent. The name is also very long so I am good with giving this a completely new name. Maybe invert the condition and name it `isShippingInitialBranch`